lib/traject_config.rb

```diff
@@ -696,7 +702,41 @@
 # Standard no.:
 #    010 XX a
 #    030 XX a
-to_field 'standard_no_display', extract_marc('010a:030a')
+to_field 'lccn_display', extract_marc('010a')
+to_field 'coden_display', extract_marc('030a')
+
+to_field 'oclc_display', extract_marc('035a') do |record, accumulator|
+  oclcs = []
+  accumulator.each_with_index do |value, i|
+    oclcs << value[7..-1] if value.start_with?('(OCoLC)')
+  end
+  accumulator.replace(oclcs)
+end
+
+to_field 'standard_no_1display' do |record, accumulator|
+  standard_no = standard_no_hash(record)
+  accumulator[0] = standard_no.to_json.to_s unless standard_no == {}
+end
+
+to_field 'lccn_param', extract_marc('010a') do |record, accumulator|
+  accumulator.each_with_index do |value, i|
+    accumulator[i] = StdNum::LCCN.normalize(value)
+  end
+end
+
+to_field 'isbn_param', extract_marc('020a') do |record, accumulator|
+  accumulator.each_with_index do |value, i|
+    accumulator[i] = ISBN.from_string(value).gsub(/\s|\-/,'')
+  end
+end
+
+to_field 'oclc_param', extract_marc('035a') do |record, accumulator|
+  oclcs = []
+  accumulator.each_with_index do |value, i|
+    oclcs << value.gsub(/\D/, '') if value.start_with?('(OCoLC)')
+  end
```

Comment:
@joycebcat will OCLC numbers appear outside of the `035 $a(OCoLC)` context that I should include in this field?